backend/systemWatcher.py

```diff
@@ -1,68 +1,69 @@
 import os
-import win32file
-import win32con
+from queue import Queue
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
ThreadPoolExecutor is decent here, but based on `engine.py` the systemWatcher already runs on a different thread.
This adds unnecessary complexity.

- Can be planned and streamlined later.